components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -595,11 +611,14 @@ abstract class AbstractFetchDownloadService : Service() {
                 val newState = download.state.copy(contentLength = download.currentBytesCopied)
                 updateDownloadState(newState)
             }
+            logger.info("verifyDownload for ${download.state.url} COMPLETED")
         }
     }
 
     private fun copyInChunks(downloadJobState: DownloadJobState, inStream: InputStream, outStream: OutputStream) {
         val data = ByteArray(CHUNK_SIZE)
+        logger.info("starting copyInChunks ${downloadJobState.state.url}" +
+                " currentBytesCopied ${downloadJobState.state.url}")
```

Comment:
You're logging the URL here twice, instead of `currentBytesCopied`.

Reply:
Thanks 👍